yb-voyager/cmd/root.go

```diff
@@ -120,3 +135,37 @@ func validateExportDirFlag() {
 		exportDir = strings.TrimRight(exportDir, "/")
 	}
 }
+
+func lockExportDir() {
+	lockfileName, err := filepath.Abs(filepath.Join(exportDir, ".lockfile.lck"))
+	if err != nil {
+		fmt.Println("Unable to create an absolute path for the export-dir lockfile")
+		os.Exit(1)
```

Comment:
You can use `utils.ErrExit(...)`.
Be concise in the messages: `"failed to get lock file path: %v", err`
Always include the `err` in the output message.